A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -528,6 +546,27 @@ class FMCMainDisplay extends BaseAirliners {
         return false;
     }
 
+    /**
+     * Any tropopause altitude up to 60,000 ft is able to be entered
+     * Format: NNNN or NNNNN Leading 0’s must be included
+     * Entry is rounded to the nearest 10 ft
+     */
```

Comment:
Might I suggest something like
```
/**
* Any tropopause altitude up to 60,000 ft is able to be entered
* @param {string | number} tropo - Format in NNNN or NNNNN leading 0's must be included. Entry is rounded to the nearest feat
* @return {boolean} - wether Tropo could be set or not
*/
```
The JSDoc will help people's IDEs identify correct type usage for parameters and return values.